test/randomized/test_transpiler_equivalence.py

```diff
@@ -37,6 +55,19 @@
     FakeSingapore,
     FakeJohannesburg,
     FakeBoeblingen,
+    FakeRochester,
+    FakeBurlington,
+    FakeCambridge,
+    FakeCambridgeAlternativeBasis,
+    FakeEssex,
+    FakeLondon,
+    FakeQasmSimulator,
+    FakeArmonk,
+    FakeRome,
+    FakeSantiago,
+    FakeSydney,
+    FakeToronto,
+    FakeValencia,
```

Comment:
It might be good to add fake washington to this list as it's a 127Q and the largest fake backend we have.

Reply:
I believe it is indeed in play via the `FakeProvider` already. This list effectively contains the backends we want to filter out for runs that require duration information (i.e. when using scheduling).

See [line 136](https://github.com/Qiskit/qiskit-terra/pull/7945/files#diff-84f51aec79a808c36c353a6ef1c3689307bf15fee8a6af356899536acadb8fb2R136).